dipy_web/urls.py

```diff
@@ -20,3 +20,6 @@
     url(r'^admin/', admin.site.urls),
     url(r'^', include('website.urls')),
 ]
+
+handler404 = 'website.views.custom404'
```

Comment:
Yes handler404 and handler500 are reserved words. Django performs some kind of magic with them and renders the views assigned to them in case of 404 or 500.